[OSDOCS#17471]Changed the references to ROSA/ROSA HCP with updated product naming in each OCM module for Classic and Rosa HCP

jheraght opened the pull request:
Version(s):
4.20+

Issue:
https://issues.redhat.com/browse/OSDOCS-17471

(Parent issue/story: https://issues.redhat.com/browse/OSDOCS-17291)

Link to docs preview:
<!--- Add direct link(s) to the exact page(s) with updated content from the preview build. --->

Peer review:
- [ ] Peer reviewer has approved this change.

QE review:
- QE approval is not required for this PR as no procedural changes.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

Additional information:
<!--- Optional: Include additional context or expand the description here.--->

<!--- After you open your PR, ask for review from the OpenShift docs team:
  For community authors: Tag @openshift/team-documentation in a GitHub comment.--->

EricPonvelle reviewed (commented):
Two small changes, but otherwise lgtm

Commit:
Resolves final error for registry docs

OSDOCS-17512:Moduralized Whats new OSD assembly.

OSDOCS-17471:Updated product names for OCM for ROSA HCP and Classic

Updated the product names

OSDOCS-17471 Update product names to use differentiated attributes

OSDOCS-17471 Update product names to use differentiated attributes

jheraght commented:
Closing this PR, made some incorrect changes, wrongly pulled into this branch, so don't want to merge this.
